### PR TITLE
Feature/circumcenter

### PR DESCRIPTION
**Pull request by josephsweeney**
Adds circumcenter function in new utils file, add matrix vector multiplication.

For issue #19

### CODE REVIEW

File: pyVor/primitives/matrix.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ def __mul__(self, other):
         if isinstance(other, (int, float)):
             return Matrix(*[Vector(*i) for i in
                             np.multiply(self._columns, other).T])
+        elif isinstance(other, (Vector)):
```
Comment (DrewMonroe) [inline pyVor/primitives/matrix.py:58]: We should write tests for this

**Review by DrewMonroe — DISMISSED** [review]

File: pyVor/primitives/matrix.py (hunk 1 above)
Comment (josephsweeney) [reply pyVor/primitives/matrix.py:58]: tests added

**Comment by Byvire** [comment]
It would be swell if you could get utils.py above 8.00/10 in Pylint. It's at 3.75 after my commit, but I can't add documentation if I can't tell what it's doing.

**Review by Byvire — APPROVED** [review]
I don't love some things but I fixed them in a different pull request anyway.

**Pull request merged**